ribs/emitters/_evolution_strategy_emitter.py

```diff
@@ -0,0 +1,187 @@
+"""Provides the EvolutionStrategyEmitter."""
+import itertools
+
+import numpy as np
+
+from ribs.emitters._emitter_base import EmitterBase
+from ribs.emitters.rankers import RankerBase
+
+
```

Comment:
I think `get_ranker` should be able to take in the following:
* A string that is a shortcut name, e.g. `rd` for `RandomDirectionRanker` -- we'll need a directory for this.
* A string that is the full name e.g. `"RandomDirectionRanker"`
* (maybe?) The class itself, e.g. `RandomDirectionRanker`, which we then instantiate
* Any other object that is an instance of `RankerBase` (check this with `isinstance`)

Furthermore, `get_ranker` should be part of the `ribs.emitters.rankers` module -- then we would just import it here and refer people to that function for available rankers.

Let me know what you think.